src/video_core/texture_cache/types.h

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "common/types.h"
+#include "video_core/texture_cache/slot_vector.h"
+
+namespace VideoCore {
+
+using ImageId = SlotId;
+using ImageViewId = SlotId;
+
+struct Offset2D {
```

Comment:
Nit, maybe it would be better to use something generic like Vec2 and Vec3 like in Citra